src/dpl/src/Graphics.h

```diff
@@ -32,35 +32,33 @@
 
 namespace dpl {
 
-using odb::dbBlock;
-using odb::dbInst;
-using odb::Point;
-using odb::Rect;
```

Comment:
is this a google style rule?

Reply:
using isn't allowed in public headers https://google.github.io/styleguide/cppguide.html#Aliases